# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.h
# Commit: d926f7fefc9d
# Full Hash: d926f7fefc9dc9eab1d2e65a87ab2883c5ef5143
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-25 21:02:33
# Regressor Bug: 1899503
# File Overlap Count: 1
# Description:
#   Bug 1899503 - P4 Update SharedWorker windowID for remote worker debugger. r=asuth
#   
#   Depends on D230260
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231778
# ==============================================================================

diff -r 01fe98c08d10 -r d926f7fefc9d dom/workers/WorkerPrivate.h
--- a/dom/workers/WorkerPrivate.h	Tue Mar 25 09:13:20 2025 +0000
+++ b/dom/workers/WorkerPrivate.h	Tue Mar 25 09:13:21 2025 +0000
@@ -1078,6 +1078,8 @@
 
   bool IsQueued() const;
 
+  void UpdateWindowIDToDebugger(const uint64_t& aWindowID, const bool& aIsAdd);
+
   already_AddRefed<WorkerRunnable> MaybeWrapAsWorkerRunnable(
       already_AddRefed<nsIRunnable> aRunnable);
 